### PR TITLE
[IOTDB-250] exclude .mvn/wrapper/maven-wrapper.jar

### PR DESCRIPTION
**Pull request by qiaojialin**

### CODE REVIEW

**Comment by justinmclean** [comment]
Not sure this fixes anything

**Comment by jixuan1989** [comment]
> Not sure this fixes anything

I think this is just for avoiding someone upload the file to the repo... No help for futures' release. RM must check their local folder carefully.

File: .gitignore [hunk 1]
```diff
@@ -79,7 +82,6 @@ grafana/data/
 
 grafana/data/test.csv
 **/lib/
-*.jar
```
Comment (jixuan1989) [inline .gitignore]: why you remove *.jar...

Comment (qiaojialin) [reply .gitignore]: it is duplicated...

**Pull request closed without merging**